onnxruntime/core/graph/schema_registry.cc

```diff
@@ -194,6 +194,23 @@ DomainToVersionMap SchemaRegistryManager::GetLatestOpsetVersions(bool is_onnx_on
   return domain_version_map;
 }
 
+static bool IsDomainVersionBeyondSupportedRange(
```

Comment:
Does it have any impact on custom ops?

Reply:
@fdwr Can you take a look at this feedback?

Reply:
It really checks anything in this registry:

`OpSchemaRegistry::DomainToVersionRange&
OpSchemaRegistry::DomainToVersionRange::Instance() {
  static DomainToVersionRange domain_to_version_range;
  return domain_to_version_range;
};`

Which includes:
"com.microsoft.nchwc", (1, 1)
"", (1, 11)
"ai.onnx.ml", (1, 2)
"com.microsoft", (1, 1)
"com.microsoft.automl", (1, 1)
"com.microsoft.dml", (1, 1)

The comment is stale, because it really applies to any intrinsically known registered domain.

`// Either all ONNX domains were within range, or the domains were not ONNX.`

->

`// Either all domains were within range, or the domains were unknown.`

Unknown domains (e.g. "foo") will return false from this function (safer), which then checks `ONNX_NAMESPACE::OpSchemaRegistry::Schema` for a match.